python/mlcroissant/mlcroissant/_src/core/uuid.py

```diff
@@ -0,0 +1,18 @@
+"""Module to manipulate UUID."""
+
+import uuid
+
+from mlcroissant._src.core.types import Json
+
+
+def generate_uid() -> str:
+    """Generates a random UUID of version 4."""
```

Comment:
Maybe you could comment on why we choose version 4?

Reply:
Done.